tool/gravity/cli/status.go

```diff
@@ -279,6 +280,11 @@ func printStatusText(cluster clusterStatus) {
 		printAgentStatus(*cluster.Agent, w)
 	}
 
+	if len(cluster.Alerts) > 0 {
+		fmt.Fprintln(w, "Cluster Alerts:")
```

Comment:
Nit: Don't want to seem too pedantic but all other sections of gravity status output capitalize only the first word so I'd rename this to `Cluster alerts:` so it looks consistent.